engine/src/universe.cpp

```diff
@@ -216,25 +216,18 @@ QSharedPointer<GenericFader> Universe::requestFader(Universe::FaderPriority prio
 
     {
         QMutexLocker fadersLocker(&m_fadersMutex);
-        if (m_faders.isEmpty())
+        for (int i = m_faders.count() - 1; i >= 0; i--)
         {
-            m_faders.append(fader);
```

Comment:
no need to this special case: If  m_faders.count()==0, then the for loop will have zero iterations, and insertPos will remain 0. insert(0, fader) in this case is an equivalent of both append and prepend, according to Qt documentation